scripts/utils/sign_and_send.js

```diff
@@ -41,10 +41,10 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
       ADDRESS_0,
       nonce
     )
-    const lightWallet = await createLightwallet()
-    const account = lightWallet.accounts[0]
-    console.log(`Signing and posting multi-send transaction request from proposer account ${account}`)
-    const sigs = signTransaction(lightWallet, [account], transactionHash)
+    const privateKey = process.env.PK
+    const account = web3.eth.accounts.privateKeyToAccount("0x" + privateKey)
```

Comment:
Since we only ever use `account.address` we could directly append `address` to the definition of `account`.